packages/coding-standards/src/Sniffs/ObjectIsCreatedByFactorySniff.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\CodingStandards\Sniffs;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+use SlevomatCodingStandard\Helpers\ClassHelper;
+use Symplify\TokenRunner\Analyzer\SnifferAnalyzer\Naming;
+
+final class ObjectIsCreatedByFactorySniff implements Sniff
+{
+    /**
+     * @var Naming
+     */
+    private $naming;
+
+    public function __construct(Naming $naming)
+    {
+        $this->naming = $naming;
+    }
+
+    /**
+     * @return int[]
+     */
+    public function register(): array
+    {
+        return [T_NEW];
+    }
+
+    /**
+     * @param File $file
+     * @param int $position
+     */
+    public function process(File $file, $position): void
+    {
+        $fileClassName = ClassHelper::getFullyQualifiedName($file, $file->findNext(T_CLASS, 2));
```

Comment:
Is there a difference in between ClassHelper::getFullyQualifiedName() and $this->naming->getClassName? Is not it possible to use only one of these services?

Reply:
ok